base/applications/rapps/gui.cpp

```diff
@@ -479,7 +479,7 @@ CMainWindow::ShowAboutDlg()
 VOID
 CMainWindow::OnCommand(WPARAM wParam, LPARAM lParam)
 {
-    BOOL bReload = TRUE;
+    const BOOL bReload = TRUE;
```

Comment:
Why not then just put TRUE whenever bReload is used in this function?

Reply:
I'm not sure. Maybe the original code does this for ease of updating in the future.